### PR TITLE
Update Controller-pairing-guide.md

### PR DESCRIPTION
**Pull request by DJm00n**
Add info about the Stadia Controller

### CODE REVIEW

**Comment by DJm00n** [comment]
Related to https://github.com/darthcloud/BlueRetro/issues/578

File: Controller-pairing-guide.md [hunk 1]
```diff
@@ -323,3 +325,17 @@ As such you need to load the **Exlene GameCube** preset.
...
    It will reconnect to BlueRetro after doing so.
+
+# 14 - Google Stadia Paring Guide
+
+## 14.1 - First pairing
+
+1. Boot up BlueRetro and make sure adapter is in inquiry mode (LED pulsing).
+2. Power up the Stadia controller by holding Stadia button + the Y button simultaneously until LED start blinking orange.
+3. LED will blink white once pairing is complete.
+
```
Comment (darthcloud) [inline Controller-pairing-guide.md:336]: Missing 14.2 how to reconnect

File: Controller-pairing-guide.md [hunk 2]
```diff
@@ -323,3 +325,17 @@ As such you need to load the **Exlene GameCube** preset.
...
+## 14.1 - Troubleshooting
```
Comment (darthcloud) [inline Controller-pairing-guide.md]: Sould be 14.3

**Review by darthcloud — CHANGES_REQUESTED** [review]

File: Controller-pairing-guide.md (hunk 1 above)
Comment (DJm00n) [reply Controller-pairing-guide.md:336]: added

**Review by darthcloud — APPROVED** [review]

**Comment by darthcloud** [comment]
Thanks for the PR!

**Pull request merged**